changelog.d/11849.misc

```diff
@@ -0,0 +1 @@
+Enable cache time-based expiry by default.
```

Comment:
It's not clear to me here that my existing Synapse installation won't start expiring caches based on time. Perhaps something like:
```suggestion
Enable cache time-based expiry by default for newly generated homeserver config files.
```